src/test/scala/com/wavesplatform/http/AddressRouteSpec.scala

```diff
@@ -35,7 +34,25 @@ class AddressRouteSpec extends RouteSpec("/addresses") with PathMockFactory with
     m
   }
 
-  private val route = AddressApiRoute(restAPISettings, wallet, state).route
+  private val consensusModule = {
+    val m = mock[WavesConsensusModule]
+    (m.generatingBalance(_: Account, _: Int)(_: TransactionModule)).expects(*,*,*).returning(0L).anyNumberOfTimes()
+    m
+  }
+
+  private val blockStorage = {
+    val m = mock[BlockStorage]
+    (m.state _).expects().returning(state).anyNumberOfTimes()
+    m
+  }
+
+  private val transactionModule = {
+    val m = mock[TransactionModule]
+    (m.blockStorage _).expects().returning(blockStorage).anyNumberOfTimes()
+    m
+  }
+
+  private val route = AddressApiRoute(restAPISettings, wallet, state, consensusModule, transactionModule).route
```

Comment:
You should not set expectations at class level unless all the tests require them. Also why is block storage mock here? If the route itself requires block storage, make the route depend on it explicitly, not implicitly via TransactionModule.